MFVNeutralino/python/EventFilter.py

```diff
@@ -128,7 +128,7 @@ def setup_event_filter(process,
                     process.load('JMTucker.Tools.UnpackedCandidateTracks_cfi')
                     process.mfvVertexTracks.tracks_src = 'jmtUnpackedCandidateTracks'
                     process.jmtRescaledTracks.tracks_src = 'jmtUnpackedCandidateTracks' # JMTBAD use rescaled tracks
-            vertexFilter = cms.EDFilter('VertexSelector', src = cms.InputTag('mfvVertices'), cut = cms.string('nTracks > 2'), filter = cms.bool(True))
+            vertexFilter = cms.EDFilter('VertexSelector', src = cms.InputTag('mfvVertices'), cut = cms.string('nTracks > 2'), filter = cms.bool(False))
```

Comment:
Is this intentional?

Reply:
Yes, otherwise there is an error in using jets in vertexer via shared-jet mitigation.

Reply:
Done.